gandalf.js

```diff
@@ -2,83 +2,85 @@
  * Extras Plugin
```

Comment:
SCRIPT-This new feature lacks unit tests. Could you add some to ensure everything works correctly?